src/test/scala/com/mozilla/telemetry/streaming/ExperimentEnrollmentsAggregatorTest.scala

```diff
@@ -0,0 +1,119 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+package com.mozilla.telemetry.streaming
+
+import java.sql.Timestamp
+import java.time.{LocalDateTime, ZoneOffset}
+
+import com.holdenkarau.spark.testing.DataFrameSuiteBase
+import org.apache.spark.sql.types._
+import org.apache.spark.sql.{DataFrame, SparkSession}
+import org.scalatest.{FlatSpec, GivenWhenThen, Matchers}
+
+class ExperimentEnrollmentsAggregatorTest extends FlatSpec with Matchers with GivenWhenThen with DataFrameSuiteBase {
+
+  val k = 10
```

Comment:
A more descriptive variable here would be nice

Reply:
Do you have any suggestions? :)
Its precise meaning varies from test case to test case. I think leaving it as constant `k` is not so bad when looking at individual tests, where we first generate  some sets of `k` messages, and then check if aggregates contain some `k`s (or its fractions).